Makefile

```diff
@@ -14,10 +14,10 @@ install: build
 	mkdir -p $(PREFIX)/bin
 	mkdir -p $(PREFIX)/lib/danger
 	cp -f $(BUILD_PATH) $(INSTALL_PATH)
+	cp -f danger-kotlin-library/build/libs/danger-kotlin-library-0.1.0-all.jar $(LIB_INSTALL_PATH)/danger-kotlin.jar
```

Comment:
I will change this to take the only item in the folder, this will avoid us to have to care about the version